frontend/package.json

```diff
@@ -12,19 +12,16 @@
 		"@highlight-run/client": "workspace:*",
 		"@highlight-run/react": "workspace:*",
 		"@highlight-run/react-mentions": "4.4.1",
-		"@highlight-run/react-select": "^5.2.2",
```

Comment:
We also have the main `react-select` package installed and are importing from that everywhere.

Reply:
Yeah, I was thinking we of doing that as a follow up. Wanted to make sure we weren't trying to install it in CI before cleaning it up.

Reply:
I did end up archiving this: https://github.com/highlight/react-select

Reply:
thank you!